projects/plugins/jetpack/modules/sso.php

```diff
@@ -206,6 +212,77 @@ public function create_error_notice_and_redirect( $query_params ) {
 		return wp_safe_redirect( $url );
 	}
 
+	/**
+	 * Render the invitation email message.
+	 */
+	public function render_invitation_email_message() {
+		// Enqueue the CSS for the admin create user page.
+		wp_enqueue_style( 'jetpack-sso-admin-create-user', plugins_url( 'modules/sso/jetpack-sso-admin-create-user.css', JETPACK__PLUGIN_FILE ), array(), time() );
+
+		$message = sprintf(
+			// translators: %s is a link to jetpack support site.
+			__( 'New users will receive an invite to join WordPress.com, so they can log in securely using %s', 'jetpack' ),
+			sprintf(
+				'<a class="jetpack-sso-admin-create-user-invite-message-link-sso" rel="noopener noreferrer" target="_blank" href="%s">%s</a>',
+				'https://jetpack.com/support/sso/',
+				__( 'Secure Sign On.', 'jetpack' )
+			)
```

Comment:
Sadly, this is still incorrect. There is no variables here at all, you can remove sprintf and put everything inside one call of __().